backend/telemetry_core/src/state/chain.rs

```diff
@@ -252,30 +253,34 @@ impl Chain {
         };
 
         if node.update_block(*block) {
-            if block.height > self.best.height {
-                self.best = *block;
-                log::debug!(
-                    "[{}] [nodes={}] new best block={}/{:?}",
-                    self.labels.best(),
-                    nodes_len,
-                    self.best.height,
-                    self.best.hash,
-                );
-                if let Some(timestamp) = self.timestamp {
-                    self.block_times.push(now.saturating_sub(timestamp));
-                    self.average_block_time = Some(self.block_times.average());
+            match block.height.cmp(&self.best.height) {
```

Comment:
I'm not a fan of this change if think the old code was easier to read.

which clippy lint is this?

Reply:
This one:
https://rust-lang.github.io/rust-clippy/master/index.html#comparison_chain